src/Server/Repository/RefreshTokenRepositoryInterface.php

```diff
@@ -18,9 +18,16 @@
 
 namespace ZfrOAuth2\Server\Repository;
 
+use ZfrOAuth2\Server\Model\RefreshToken;
+
 /**
  * Interface RefreshTokenRepositoryInterface
  */
 interface RefreshTokenRepositoryInterface extends TokenRepositoryInterface
```

Comment:
Never mind, I understand the reasn: it allows you tos implify the TokenService class by having the same constructor.